scripts/lib/LinkChecker.ts

```diff
@@ -117,6 +117,52 @@ export class Link {
     );
   }
 
+  /**
+   * Returns a string with a suggested replacement for a broken link
+   */
+  didYouMean(existingFiles: File[], originFile: string): string {
+    const levenshtein = require("fast-levenshtein");
+
+    // Find a new valid link
+    let min_score = Number.MAX_SAFE_INTEGER;
+    let suggestion_path: String = "";
+    let suggestion_path_anchors: String[] = [];
+
+    const possiblePaths = this.possibleFilePaths(originFile);
+    const pathNoExtension = possiblePaths[0].replace(/\.[^\/.]+$/, "");
+
+    existingFiles.forEach((file) => {
+      let score = levenshtein.get(pathNoExtension, file.path);
+      if (score < min_score) {
```

Comment:
Does this mean we always no matter what give a suggestion, since MAX_SAFE_INTEGER is so high? Generally people recommend only offering a suggestion if you're within a certain distance. Otherwise the suggestion ends up being not at all relevant and is noisy.

Reply:
Hmm.. you are right! We shouldn't suggest a link that it's not close to the broken one. Instead of modifying MAX_SAFE_INTEGER, I normalized the final score between 0 and 1, to check that we have at least 0.5 of similarity. We can change the 0.5 using the MIN_SIMILARITY constant at the beginning. What do you think?